web/src/helpers.tsx

```diff
@@ -20,11 +20,17 @@ import { MenuDataItem } from '@ant-design/pro-layout';
 import { history } from 'umi';
 import moment from 'moment';
 
+import InfoCircleOutlined from '@ant-design/icons';
```

Comment:
1. ![image](https://user-images.githubusercontent.com/2106987/101305289-4daa3580-387d-11eb-8471-db0291cdbaef.png)

2. You may use the wrong import method

```ts
import {InfoCircleOutlined} from '@ant-design/icons';
```